docs/components/camera/esp32-camera.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Configure an esp32-camera (viam-micro-server)"
+linkTitle: "esp32-camera"
+weight: 33
+type: "docs"
+description: "Configure a camera connected to an esp32 board, initialized and configured using esp-idf."
+images: ["/icons/components/camera.svg"]
+tags: ["camera", "components", "micro-RDK"]
+component_description: "An `OV2640` or `OV3660` camera connected to an esp32 board."
+micrordk_component: true
+toc_hide: true
+# SMEs: Matt Perez, micro-RDK team
+---
+
+`esp32-camera` is the camera model that supports all cameras that work with Espressif's [esp32-camera drivers](https://github.com/espressif/esp32-camera) including:
+
+- `OV2640`
+
+  - [Datasheet](https://www.uctronics.com/download/OV2640_DS.pdf)
+  - 1600 x 1200, Color, ¼” lens
+  - You can use a cam ribbon adapter to connect to your `esp32` board
+
+- `OV3660`: an [m5 camera timer module](https://docs.m5stack.com/en/unit/timercam_f)
+  - [Datasheet](https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/unit/OV3660_CSP3_DS_1.3_sida.pdf)
+  - 2048 x 1536, Color, ⅕” lens
+
+For example, the `OV2640` with a ribbon cable connected to an ESP32:
+
+{{< imgproc src="/components/camera/esp32-camera-2640.png" alt="Fake Camera on the ESP32" resize="300x" >}}
+
+{{< alert title="Software requirements" color="note" >}}
+To use this model, you must follow the [Micro-RDK Development Setup](/installation/micro-rdk-dev/), which enables you to install and activate the ESP-IDF.
+At the step [Generate a new project from the micro-RDK template](/installation/micro-rdk-dev/#generate-a-new-project-from-the-micro-rdk-template) where you create a new project with `cargo generate`, select the option to include camera module traits when prompted.
+Finish the [Micro-RDK Development Setup](/installation/micro-rdk-dev/) and return to this guide.
+{{< /alert >}}
+
+{{< alert title="Info" color="info" >}}
+
+The`esp32-camera` camera model is not currently available as a built-in option in [the Viam app](https://app.viam.com), so you cannot use **Builder** mode to configure this camera.
+
+{{< /alert >}}
+
+{{< tabs name="Configure a esp32-camera" >}}
+{{% tab name="JSON Template" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Select **Builder** mode.
+Copy and paste the following JSON into your existing machine configuration in the `"components"` level:
```

Comment:
This doesn't make sense

Reply:
ah yes it needs to be JSON mode, changed